packages/scan/src/requests/GetScan.ts

```diff
@@ -0,0 +1,13 @@
+import { ScanState } from '../Scans';
+import { HttpOptions, HttpRequest } from '@secbox/bus';
+
+export class GetScan extends HttpRequest<void, ScanState> {
```

Comment:
`void` makes sense only in return types

```suggestion
export class GetScan extends HttpRequest<undefined, ScanState> {
```